### PR TITLE
Fix spelling errors

### PR DESCRIPTION
**Pull request by ElyesH**
Signed-off-by: Elyes Haouas <ehaouas@noos.fr>

### CODE REVIEW

**Comment by acpibob** [comment]
I have a problem with replacing "unsecure" with "insecure".

Definition of "unsecure":

(especially of a computer system) not protected against attack or other criminal activity.
"an unsecure WiFi network"

**Comment by rafaeljw** [comment]
I agree.

@ElyesH : Please drop the "unsecure" -> "insecure" change and resubmit.

File: source/common/dmtbinfo2.c [hunk 1]
```diff
@@ -2030,7 +2030,7 @@ ACPI_DMTABLE_INFO           AcpiDmTableInfoSdevHdr[] =
 {
     {ACPI_DMT_SDEV,     ACPI_SDEVH_OFFSET (Type),                   "Subtable Type", 0},
     {ACPI_DMT_UINT8,    ACPI_SDEVH_OFFSET (Flags),                  "Flags (decoded below)", 0},
-    {ACPI_DMT_FLAG0,    ACPI_SDEVH_FLAG_OFFSET (Flags,0),           "Allow handoff to unsecure OS", 0},
```
Comment (rafaeljw) [inline source/common/dmtbinfo2.c:2033]: Please drop this one.

**Review by ElyesH — COMMENTED** [review]
Thx

**Pull request merged**